src/Scrapers/Scraper.py

```diff
@@ -10,4 +12,12 @@ def __init__(self, url) -> None:
 
     @abstractmethod
     def get_problem(self):
-        pass
\ No newline at end of file
+        pass
+    
+    def get_proxy(self):
+        config_file_path = config_file_path = Path(os.path.dirname(os.path.realpath(__file__)), '../Config/config.json')
```

Comment:
Kindly use the function already provided in the Config class for getting config_file_path. Its called get_config_path.